src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    private String description;
+    private boolean marking;
```

Comment:
boolean values should be named to sound like booleans, consider "isMarked" or "isFinished"